src/main.rs

```diff
@@ -102,7 +108,11 @@ fn main() -> anyhow::Result<()> {
         bail!("Some packages failed to be checked.");
     }
 
-    println!("{success}{bold}Feature combination checks successful! Congrats :){reset}");
+    if cli.dry_run {
+        println!("Dry run completed, no checks were run.")
```

Comment:
```suggestion
        println!("{info}{bold}Dry run completed, no checks were run.{reset}")
```

(Nit) We can make this message more eye-catching by making it blue.